detector_ros/config/param.yaml

```diff
@@ -0,0 +1,10 @@
+h_min: 100
+s_min: 110
+v_min: 110
+h_max: 179
+s_max: 255
```

Comment:
if possible, add a short description of the params. have a look at the planning library param files.